Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM       node:8.11.0
+FROM       node:8.11.1-stretch
 LABEL maintainer="Automattic"
 
+RUN echo "deb http://ftp.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list
+RUN apt-get update
+RUN apt-get install -t stretch-backports zopfli
+RUN apt-get install -t stretch-backports brotli
```

Comment:
I sense that this is still draft material, but it would be good to merge these into a single line due to the way Docker creates a separate layer for each command. further, there should be no need to run `apt-get` twice, right?

```Dockerfile
RUN echo "deb http://ftp.debian.org/debian …" >> /etc/apt/sources.list.d/backports.list &&
    apt-get update &&
    apt-get install -t street-backports zopfli brotli
```

Reply:
yup, very drafty. the jesse image that the plain node image is based on doesn't support brotli at all and only an old version of zopfli. This was the result of me playing around trying to determine how we could get them into the image.